### PR TITLE
removing disconnect & enter app button from header

### PR DESCRIPTION
**Pull request by mitchtrale**

### CODE REVIEW

**Review by Ely-S — APPROVED** [review]
works great!

**Comment by ryanlabouve** [comment]
@mitchtrale plz fix warnings

File: app/components/Header.tsx [hunk 1]
```diff
@@ -23,18 +23,6 @@ export default function Header() {
...
-            <button
-              style={{
-                background: "linear-gradient(270deg, #1BD6CF 0%, #00E5AF 100%)",
-              }}
-              className="bg-brand-green text-gray-900 py-2 px-8 rounded-md text-sm"
-            >
-              <Link to="/app">Enter App</Link>
-            </button>
-          )}
-        </div>
```
Comment (johnmosesman) [inline app/components/Header.tsx:37]: Why's this deleted?

**Comment by ryanlabouve** [comment]
@mitchtrale we may consider making `app` the main page of the app in this PR in prep of the `app.` subdomain and remove the current index

**Pull request merged**